### PR TITLE
DOC: cleanup 1.0.2 whatsnew

### PR DESCRIPTION
**Pull request by TomAugspurger**

### CODE REVIEW

File: doc/source/whatsnew/v1.0.2.rst [hunk 1]
```diff
@@ -18,13 +18,13 @@ Fixed regressions
 - Fixed regression in :meth:`DataFrame.to_excel` when ``columns`` kwarg is passed (:issue:`31677`)
 - Fixed regression in :meth:`Series.align` when ``other`` is a DataFrame and ``method`` is not None (:issue:`31785`)
 - Fixed regression in :meth:`pandas.core.groupby.RollingGroupby.apply` where the ``raw`` parameter was ignored (:issue:`31754`)
-- Fixed regression in :meth:`rolling(..).corr() <pandas.core.window.Rolling.corr>` when using a time offset (:issue:`31789`)
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v1.0.2.rst]: Why did you remove this? (I personally think it gives a nicer link)

The link target is actually also wrong, I see, since we moved Rolling into a rolling file (will do a PR to fix this)

Comment (TomAugspurger) [reply doc/source/whatsnew/v1.0.2.rst]: Ah, I just assumed the `< ... >` syntax didn't work with :meth:. If that's an option then I'd prefer it to `core.window.Rolling`.
